HIP/hip-786.md

```diff
@@ -108,14 +108,14 @@ message NodeStakeUpdateTransactionBody {
    * The total tinybars to be paid as staking rewards in the ending period, after applying
    * the settings for the 0.0.800 balance threshold and the maximum stake rewarded.
    */
-  int64 total_reward = 14;
+  int64 max_total_reward = 14;
 }
 ``` 
 
 ### Mirror nodes
 
 If a mirror node API exposes network staking properties, it _should_ expand to include this new information
-and _should_ replace the `staking_reward_rate` field with `total_reward`. For example,
+and _should_ replace the `staking_reward_rate` field with `max_total_reward`. For example,
```

Comment:
We can't remove `staking_reward_rate` as it would break backwards compatibility plus it was populated for historical data. We should remove the `replace` wording and just mention we're adding new fields and consumers should update to use the new field going forward. Similarly, we should remove the `remove` wording from the REST response below.